whitelist_python.py

```diff
@@ -0,0 +1,3 @@
+import collections
+
+collections.defaultdict(list).default_factory
```

Comment:
But, if we add `collections.defaultdict(list).defaultfactory`, then vulture won't show this as dead code even if we don't supply the whitelist file - as it would see that the user has not just left the deafult_factory set to None, but has used it later - contrary to what happened earlier, where it would report: ` whitelist_python.py:3: Unused attribute 'default_factory ` .

Am I wrong?